packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -88,11 +89,16 @@ class AddressDetails extends React.Component {
                             ? getLocation(this.props.states_list, this.state.address_state_to_display, 'value')
                             : getLocation(this.props.states_list, values.address_state, 'value');
                     }
-                    this.props.onSubmit(this.props.index, values, actions.setSubmitting);
+                    this.props.onSubmit(
+                        this.props.getCurrentStep() - 1,
```

Comment:
is it possible to implement a "`getPreviousStep()`" method for better readability?

Reply:
In fact this is `getCurrentStep` but since the `state.items` in `Account Wizard` in an array and it starts from 0, i need to use -1 to get the correct array index.